tools/install.sh

```diff
@@ -23,7 +23,27 @@ main() {
   # Only enable exit-on-error after the non-critical colorization stuff,
   # which may fail on systems lacking tput or terminfo
   set -e
+}
+
+check_optional_args() {
+  # Set default values
+  LAUNCH_ZSH_AFTER=1
+
+  # Evaluate arguments
+  for arg in $@; do
+    case "$arg" in
+      --batch)
+        LAUNCH_ZSH_AFTER=0
+        printf "${BLUE}--batch:${NORMAL} zsh will not be started after installation\n"
```

Comment:
This breaks when `${BLUE}` is an empty string (e.g., Docker environment). In such a case `--` is interpreted as argument separator and the command fails.

Please replace it by the following line to avoid the issue, or just don't write the `--batch ` at the beginning of the string:
```
        printf "%s\n" "${BLUE}--batch:${NORMAL} zsh will not be started after installation\n"
```